cmd/variables.go

```diff
@@ -63,6 +63,12 @@ func (h *Handler) VariablesSet(ctx context.Context, req *entity.CommandRequest)
 		skipRedeploy = false
 	}
 
+	replace, err := req.Cmd.Flags().GetBool("replace")
+	if err != nil {
+		// The flag is optional; default to false.
+		replace = false
+	}
```

Comment:
Thoughts on having a prompt that says "WARNING YOU'RE REPLACING ALL YOUR VARIABLES! ONLY DO THIS IF YOU KNOW WHAT YOU'RE DOING!"

Reply:
Like ideally would love a "--replace --yes" flag